desktop/src/gui/controller.rs

```diff
@@ -511,46 +509,145 @@ fn try_wgpu_backend(backend: wgpu::Backends) -> Option<wgpu::Instance> {
     }
 }
 
-// try to load known unicode supporting fonts to draw cjk characters in egui
+// Load fallback fonts
 fn load_system_fonts(
     font_database: &Database,
-    locale: LanguageIdentifier,
-) -> anyhow::Result<egui::FontDefinitions> {
-    let mut families = Vec::new();
-    if let Some(windows_font) = match locale.language.as_str() {
-        "ja" => Some(Family::Name("MS UI Gothic")),
-        "zh" => Some(match locale.to_string().as_str() {
-            "zh-CN" => Family::Name("Microsoft YaHei"),
-            _ => Family::Name("Microsoft JhengHei"),
-        }),
-        "ko" => Some(Family::Name("Malgun Gothic")),
-        _ => None,
-    } {
-        families.push(windows_font);
+    locale: unic_langid::LanguageIdentifier,
+) -> egui::FontDefinitions {
+    let mut fd: FontDefinitions = egui::FontDefinitions::default();
+
+    let lang = locale.language.as_str();
+    let is_ja = lang == "ja";
```

Comment:
Would probably be simpler with `langid!`

Reply:
What do you mean? `langid!` would check all locale properties, we're interested in language only

Reply:
Ah you are right, I had somehow thought that `locale == langid!("ko")` would just match the language.